sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -49,6 +49,7 @@ object Klaviyo {
      *
      * @param applicationContext
      */
+    @JvmStatic
```

Comment:
`@JvmStatic` allows the Java dev to call 
```
Klaviyo.initialize()
```

instead of 
```
Klaviyo.INSTANCE.initialize()
```

it is an additive change, so this doesn't break for folks currently using `INSTANCE`